test/notifications_test.dart

```diff
@@ -166,11 +167,24 @@ void main() {
         expectedTagComponent: 'stream:${message.streamId}:${message.subject}');
     }));
 
-    test('group DM', () => awaitFakeAsync((async) async {
+    test('group DM: 2 users', () => awaitFakeAsync((async) async {
       await init();
+      final zulipLocalizations = GlobalLocalizations.zulipLocalizations;
       final message = eg.dmMessage(from: eg.thirdUser, to: [eg.otherUser, eg.selfUser]);
       await checkNotifications(async, messageFcmMessage(message),
-        expectedTitle: "${eg.thirdUser.fullName} to you and 1 others",
+        expectedTitle: zulipLocalizations.groupDMNotificationMessage(
+          eg.thirdUser.fullName, message.allRecipientIds.length - 2),
```

Comment:
Given that the point of this test case is to check that we get the singular right, it's better to have the expected text be explicit here. It means that if the text changes, we'll need to update the test, but that's worth it when the text is the focus of the test case.

Reply:
Okay. Fixed.